audio-worklet/src/voice.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+class Voice {
+  /**
+   * A voice generates a waveform and filters it, exposing the sound through
+   * this.output.
+   * @param {BaseAudioContext} context the audio context
+   * @param {Number} frequency the frequency corresponding to the voice's note
+   * @param {Object} options optional parameters
+   * @param {String} options.waveform the oscillator's waveform
+   * @param {Number} options.lowPassCutoff the lowpass filter frequency
+   * @param {Number} options.attack milliseconds until full amplitude
+   * @param {Number} options.decay milliseconds until sustain level
+   * @param {Number} options.sustain the steady amplitude of the note as it is
+   *                                 pressed
+   * @param {Number} options.release milliseconds between the release of a note
+   *                                 and zero amplitude
+   */
+  constructor(context, frequency, options) {
+    this.context_ = context;
+    let waveform = options.waveform || 'sawtooth';
+    let lowPassCutoff_ = options.lowPassCutoff || 5000;
```

Comment:
Again why 5000?  And shouldn't this be synchronized somehow in all the other places you have a magic 5000? Are they supposed to be the same?